src/Tizen.NUI.Components/Controls/Loading.cs

```diff
@@ -114,6 +116,7 @@ public Loading(string style) : base(style)
         /// <since_tizen> 8 </since_tizen>
         public Loading(LoadingStyle loadingStyle) : base(loadingStyle)
         {
+            Initialize();
```

Comment:
please check the warning below. => line 135 "Properties should not return arrays"
returning array is not good in public method, if there is not other way, you may use read only collection as return type.
reference : https://docs.microsoft.com/ko-kr/dotnet/fundamentals/code-analysis/quality-rules/ca1819

Reply:
The warning mentioned concerns code that wasn't changed in this pull request.